automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/library/DataLibraryPage.scala

```diff
@@ -97,7 +97,7 @@ class DataLibraryPage(implicit webDriver: WebDriver) extends BaseFireCloudPage
     libraryTable.getRows
   }
 
-  case class RequestAccessModal(override implicit val webDriver: WebDriver) extends MessageModal {
+  case class RequestAccessModal() extends MessageModal {
```

Comment:
what was the inspection for this change? was it that the val is already defined in the superclass?